README.md

```diff
@@ -21,3 +45,80 @@ Pi is roughly 3.138040
 
 in the output.
 
+## Using Redis for Caching
+
+1. Connect to the spark-user container:
+   ```bash
+   docker compose exec -it spark-user bash
+   ```
+
+2. Create a Python file with the example code:
+   ```bash
+   cat > redis_example.py << 'EOF'
```

Comment:
I would suggest just adding this file in the repo somewhere and volume mounting it into spark-user, since that container is just for testing anyway

Reply:
👍